Tasks/Gradle/task.json

```diff
@@ -342,6 +342,8 @@
         "sqAnalysis_IsPullRequest_SkippingBuildBreaker": "Pull request build: build will not be broken if quality gate fails.",
         "sqAnalysis_BuildBrokenDueToQualityGateFailure": "The SonarQube quality gate associated with this build has failed.",
         "sqAnalysis_QualityGatePassed": "The SonarQube quality gate associated with this build has passed (status %s)",
+        "sqAnalysis_UnknownComparatorString": "Unknown operator string: %s",
```

Comment:
Not sure if this is helping the user in any way. A more general error message like "There was a problem creating the SonarQube summary - unkown operator %s"